AnimalTraits.java

```diff
@@ -0,0 +1,13 @@
+public abstract interface AnimalTraits{
```

Comment:
By convention interfaces are named `ISomething`: starts with a capital **'I'**, followed by the **ability**.  Eg.
```java
IDamageable
IFightable
ISortable
IDescribeable
IDisplayable
// etc.
```